include/swift/SwiftRemoteMirror/SwiftRemoteMirrorTypes.h

```diff
@@ -77,6 +77,7 @@ typedef struct swift_reflection_info {
   swift_reflection_section_pair_t capture;
   swift_reflection_section_pair_t type_references;
   swift_reflection_section_pair_t reflection_strings;
+  swift_reflection_section_pair_t multipayload_enums; // TODO: Is this okay?
```

Comment:
Q: Is it okay to just add another field here?  Does that mess up existing clients?  Is there an alternative?

Reply:
@mikeash Do you know what the consequences here are of adding another field for a new section?

Reply:
Thanks for clarifying.  I've backed out this change.